core/src/entities/packages/Packages.md

```diff
@@ -0,0 +1,76 @@
+## TODO Update Diagram
+
+```mermaid
```

Comment:
I love Mermaid....  Do you still need that TODO above?

Reply:
yeah or we need to delete it.  This is woefully out of date.

Reply:
I've got an action item to update this. I think I'll take a pass at what it will look like if we use Component instead of packages like we talked about.